Tasks/SonarQubePostTest/Microsoft.VisualStudio.Services.CodeAnalysis.PostProcessor.dll.config

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not sure this config file is needed. I will test and remove it if not required.